test/target-branch.test.js

```
@@ -0,0 +1,46 @@
+import {
+  getDefaultBranch,
+  getRemoteBranches,
+  createPR
+} from '../index.js';
```

ESLint

[error] forbid file extension '.js'.

(n/file-extension-in-import)